.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.DS_Store
```

Comment:
You need to reinstate the original `.gitignore` that was created for you when you first built the app.

Reply:
It wasn't created when we originally ran react create app, but I think that's because we didn't use npx in front of the command. So I created another React app in a different folder to grab what was auto created in the .gitignore and then added it to our .gitignore file. I know that's probably not best practice so just let us know if you would rather we re-build the app.